src/libsyntax/codemap.rs

```diff
@@ -103,18 +103,11 @@ impl FileLoader for RealFileLoader {
 //
 
 pub struct CodeMap {
-    // The `files` field should not be visible outside of libsyntax so that we
-    // can do proper dependency tracking.
     pub(super) files: RefCell<Vec<Rc<FileMap>>>,
```

Comment:
Yes, there is one instance in libsyntax where it is accessed via `borrow_mut()`.